proto/frequenz/api/weather/weather.proto

```diff
@@ -71,19 +77,18 @@ message ForecastRequest {
   frequenz.api.common.location.Location location = 1;
 
   // List of required features.  If none are specified, all available features
-  // will be streamed.
+  // will be streamed., if it is used it will be ignored.
   repeated ForecastFeature features = 2;
 
+  // Optional for StreamLiveWeatherForecast, ignored if provided.
   // The start of the period for which the forecast is being requested.
```

Comment:
Yes, so since the service is split into two rpcs, one for historicalforecasts one for streaming, the start_ts and end_ts are only used for the historical part. For streaming they are not necessary and if they are provided for that RPC they are ignored. That's why the the second part explaining what happens when they are left out is removed.